examples/with-temporal/components/ListItem.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Link from 'next/link'
+
+import { User } from '../interfaces'
+
+type Props = {
+  data: User
+}
+
+const ListItem = ({ data }: Props) => (
+  <Link href="/users/[id]" as={`/users/${data.id}`}>
```

Comment:
```suggestion
  <Link href={`/users/${data.id}`}>
```

`as` is no longer needed 😄